### DISCUSSION TITLE
Ghost cells in a linear system solve

### DISCUSSION THREAD

**Discussion by ZiyangHuang** [AMReX-Codes/amrex#3367] [Q&A] [answered]
I am using the "MLABecLaplacian" to solve my problem that has either periodic or homogeneous Neumann BC. Does the "multifab" that stores the solution need any ghost cells? Thanks!

**Comment by WeiqunZhang** [comment, marked as answer]
No, the solution MultiFab is not required to have ghost cells. If it does not have exactly one ghost cell, the solver will create a temporary internally. So for performance, it might be slightly better to have one ghost cell.
